blueprints/secops/bindplane-gke/config/values.yaml.tpl

```diff
@@ -0,0 +1,89 @@
+# This ingress example uses Ingress NGINX and Cert Manager
+# for creating Lets Encrypt signed certificates.
+#
+# - https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke
+# - https://cert-manager.io/docs/tutorials/getting-started-with-cert-manager-on-google-kubernetes-engine-using-lets-encrypt-for-ingress-ssl/
+#
+ingress:
+  enable: true
+  host: ${hostname}
+  class: "gce-internal"
+  tls:
+    enable: true
+    secret: bindplane-tls
+  annotations:
+    # cert-manager.io/issuer: letsencrypt
+    kubernetes.io/ingress.regional-static-ip-name: ${address}
+
+config:
+  # Use the secret named "bindplane", which contains
+  # the license, username, password, secret_key, and sessions_secret.
+  # If you do not want to use a secret, see the comment below and
+  # disable this option.
+  licenseUseSecret: true
+
+  # Defaults to wss://bindplane.bindplane.svc.cluster.local:3001/v1/opamp,
+  # which is the bindplane namespace's bindplane service. This is suitable
+  # for connecting agents within the same cluster. We are using ingress
+  # so server_url needs to be updated to the ingress host.
+  # NOTE: server_url maps to bindplane's network.remoteURL option.
+  server_url: https://${hostname}
+
+
+# Enables mutli account, allowing you to create
+# multiple Tenants within the same BindPlane instance.
+multiAccount: true
```

Comment:
This is a legacy option and can be safely removed. I understand it is in the upstream values and will make a ticket on our (observIQ) end to clean it up.

Reply:
Got rid of that, was just inherited from the official chart repository!